src/main/java/org/alliancegenome/curation_api/services/AllelePhenotypeAnnotationService.java

```diff
@@ -0,0 +1,95 @@
+package org.alliancegenome.curation_api.services;
+
+import java.util.List;
+
+import org.alliancegenome.curation_api.dao.AllelePhenotypeAnnotationDAO;
+import org.alliancegenome.curation_api.dao.ConditionRelationDAO;
+import org.alliancegenome.curation_api.enums.BackendBulkDataProvider;
+import org.alliancegenome.curation_api.exceptions.ObjectUpdateException;
+import org.alliancegenome.curation_api.model.entities.AllelePhenotypeAnnotation;
+import org.alliancegenome.curation_api.model.entities.Allele;
+import org.alliancegenome.curation_api.model.ingest.dto.fms.PhenotypeFmsDTO;
+import org.alliancegenome.curation_api.response.ObjectResponse;
+import org.alliancegenome.curation_api.response.SearchResponse;
+import org.alliancegenome.curation_api.services.base.BaseEntityCrudService;
+import org.alliancegenome.curation_api.services.validation.dto.fms.AllelePhenotypeAnnotationFmsDTOValidator;
+
+import jakarta.annotation.PostConstruct;
+import jakarta.enterprise.context.RequestScoped;
+import jakarta.inject.Inject;
+import jakarta.transaction.Transactional;
+
+@RequestScoped
+public class AllelePhenotypeAnnotationService extends BaseEntityCrudService<AllelePhenotypeAnnotation, AllelePhenotypeAnnotationDAO> {
+    
+    @Inject
+	AllelePhenotypeAnnotationDAO allelePhenotypeAnnotationDAO;
+	@Inject
+	ConditionRelationDAO conditionRelationDAO;
+	@Inject
+	PhenotypeAnnotationService phenotypeAnnotationService;
+	@Inject
+	AllelePhenotypeAnnotationFmsDTOValidator allelePhenotypeAnnotationFmsDtoValidator;
+
+	@Override
+	@PostConstruct
+	protected void init() {
+		setSQLDao(allelePhenotypeAnnotationDAO);
+	}
+
+	public ObjectResponse<AllelePhenotypeAnnotation> get(String identifier) {
+		SearchResponse<AllelePhenotypeAnnotation> ret = findByField("curie", identifier);
+		if (ret != null && ret.getTotalResults() == 1)
+			return new ObjectResponse<AllelePhenotypeAnnotation>(ret.getResults().get(0));
+		
+		ret = findByField("modEntityId", identifier);
+		if (ret != null && ret.getTotalResults() == 1)
+			return new ObjectResponse<AllelePhenotypeAnnotation>(ret.getResults().get(0));
+		
+		ret = findByField("modInternalId", identifier);
+		if (ret != null && ret.getTotalResults() == 1)
+			return new ObjectResponse<>(ret.getResults().get(0));
+		
+		ret = findByField("uniqueId", identifier);
+		if (ret != null && ret.getTotalResults() == 1)
+			return new ObjectResponse<>(ret.getResults().get(0));
+				
+		return new ObjectResponse<>();
+	}
+
+	@Override
+	@Transactional
+	public ObjectResponse<AllelePhenotypeAnnotation> update(AllelePhenotypeAnnotation uiEntity) {
+		return null;
+	}
+
+	@Override
+	@Transactional
+	public ObjectResponse<AllelePhenotypeAnnotation> create(AllelePhenotypeAnnotation uiEntity) {
```

Comment:
i would leave these methods out if there is no implementation or throw a not implemented exception